dcm4che-net/src/main/java/org/dcm4che3/net/Device.java

```diff
@@ -1084,7 +1063,11 @@ public ApplicationEntity getApplicationEntity(String aet) {
      * @return AE titles of this device, including alias AE titles
      */
     public Collection<String> getApplicationAETitles() {
-        return aliasApplicationEntitiesMap.keySet();
+        HashSet<String> aeTitles = new HashSet<>();
+        aeTitles.addAll(applicationEntitiesMap.keySet());
+        //add alias AE titles
```

Comment:
Nit-pick: Not sure if the comment adds any value, we can see that by the one liner and the public method comment mentions that too.

Reply:
should be ok